src/Symfony/Component/Routing/Loader/DependencyInjection/ServiceRouterLoaderContainer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing\Loader\DependencyInjection;
+
+use Psr\Container\ContainerInterface;
+
+/**
+ * @internal
+ */
+class ServiceRouterLoaderContainer implements ContainerInterface
+{
+    private $container;
+    private $serviceLocator;
+    private $serviceRouterLoaderTag;
+
+    public function __construct(ContainerInterface $container, ContainerInterface $serviceLocator, string $serviceRouterLoaderTag)
+    {
+        $this->container = $container;
+        $this->serviceLocator = $serviceLocator;
+        $this->serviceRouterLoaderTag = $serviceRouterLoaderTag;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function get($id)
+    {
+        if ($this->serviceLocator->has($id)) {
+            return $this->serviceLocator->get($id);
+        }
+
+        @trigger_error(sprintf('Not tagging the "%s" service with the "%s" tag is deprecated since Symfony 4.3.', $id, $this->serviceRouterLoaderTag), E_USER_DEPRECATED);
```

Comment:
Registering a routing loader without tagging it... and will be required in 5.0.

Reply:
Registering the routing loader "%s" without tagging it with the "%s" tag is deprecated since Symfony 4.3 and will be required in Symfony 5.0. ?

Or the service id won't be visible anymore.

Reply:
sure